api/kessel/relations/v1beta1/relation_tuples.proto

```diff
@@ -34,6 +34,21 @@ service KesselTupleService {
 			delete: "/v1beta1/tuples"
 		};
 	};
+	rpc ImportBulkTuples (stream
+			ImportBulkTuplesRequest) returns (ImportBulkTuplesResponse) {
+		option (google.api.http) = {
+			post: "/v1beta1/tuples/bulkimport"
+			body: "*"
+		};
+	};
+}
+
+message ImportBulkTuplesRequest {
+	repeated Relationship tuples = 1;
```

Comment:
Keeping inline with SpiceDB I think it makes sense to have a validation rule on `tuples` like it is done for relationships here: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.ImportBulkRelationshipsRequest

Reply:
updated